### PR TITLE
Fargate tut

### PR DESCRIPTION
**Pull request by Xantier**
Preview url: https://602d000fdbcf050007fea389--roadie.netlify.app/blog/backstage-fargate-up-and-running/

### CODE REVIEW

**Comment by Xantier** [comment]
We can probably add a Roadie plug to the end of the post.

File: content/blog/backstage-fargate-up-and-running/index.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+
+## Step 4 - Creating a Load Balancer for our Backstage service
+
+The last scaffolding bit we want to do to support our Fargate Backstage is to set up a load balancer in standby to wait for our Fargate service to attach itself to it. We do this step a bit prematurely just to have a good static URL available to point to when we eventually start building the actual Backstage application. 
+
+Let's navigate to [AWS Load Balancer section](https://eu-west-1.console.aws.amazon.com/ec2/v2/home?region=eu-west-1#LoadBalancers) in the console and spin one up. 
+
+We want to create an Application Load balancer that is internet facing. It is a good idea to select all subnets, so the load balancer is able to target our containers even if they are spread out across different AWS Availability Zones. For now the only listener we attach to the load balancer is listening to HTTP traffic through port 80, but if you have a domain that you control and can create certificates for, you should be using HTTPS and port 443.  
+![lb_basic_setup.png](./lb_basic_setup.png)
```
Comment (dtuite) [inline content/blog/backstage-fargate-up-and-running/index.md:170]: This needs a newline just above. Without it the image is squashed up against the paragraph above.

File: content/blog/backstage-fargate-up-and-running/index.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+![fargate_provisioning_tasks.png](./fargate_provisioning_tasks.png)
+
+It will take few minutes before it reaches 'RUNNING' status. Unfortunately it doesn't seem to stay in 'RUNNING' status for too long and instead ends up in a loop of starting a new task and failing one after another. 
+
+We can investigate and debug why the running container fails to stay up by checking [CloudWatch logs](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logsV2:log-groups) that our task has written. In cases where the task doesn't start at all we can take a look at the task itself from the ECS pages to see what prevents it from starting. These could be something like IAM policy issues or perhaps a wrong URL to the image that we have defined.
+
+When we take a look at the logs we can see that the container starts up and Backstage itself within the container tries its best to start up. It fails on Knex timeout error, telling us that Knex is unable to connect to the database.
+![db_connectivity_error.png](./db_connectivity_error.png)
```
Comment (dtuite) [inline content/blog/backstage-fargate-up-and-running/index.md:334]: Another missing newline.

**Review by dtuite — CHANGES_REQUESTED** [review]
I flagged two missing newlines. Other than that, I think you've raised the bar for technical tutorials on Roadie. This is incredible!

**Review by dtuite — APPROVED** [review]

**Comment by dtuite** [comment]
@Xantier feel free to merge this whenever you like. It will be deployed automatically when it is merged.

**Pull request merged**